contracts/interfaces/ICover.sol

```diff
@@ -9,27 +9,44 @@ interface ICover is IERC721 {
   /* ========== DATA STRUCTURES ========== */
 
   struct StakingPool {
-    uint160 id;
-    uint96 bookedAmount;
+    address poolAddress;
+    uint96 coverAmount;
+    uint96 premiumInNXM;
   }
 
   struct Cover {
```

Comment:
The contract name and the struct name use the same name. This is a bit confusing and also confuses the autocomplete in the IDE. Let's rename one of them.